packages/core/cli.ts

```diff
@@ -138,8 +138,13 @@ yargs(process.argv.slice(2))
 		},
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		(args: any) => {
+			const fT = path.join(process.cwd(), args.fuzzTarget);
 			const opts: Options = {
-				fuzzTarget: path.join(process.cwd(), args.fuzzTarget),
+				fuzzTarget: [".js", ".mjs", ".cjs"].some((suffix) =>
```

Comment:
Extract this functionality out into a dedicated method so that it can be reused for `customHooks` as well?

Reply:
fair point. done